qemu/tests/cpu_model_inter_generation.py

```diff
@@ -0,0 +1,60 @@
+import logging
+
+from virttest import error_context
+from virttest import env_process
+from virttest.libvirt_xml import domcapability_xml
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Boot latest cpu model on old host
+
+    :param test: QEMU test object.
+    :param params: Dictionary with test parameters.
+    :param env: Dictionary with the test environment.
+    """
+
+    def start_with_model(test_model):
+        """
+        Start vm with tested model
+        :param test_model: The model been tested
+        """
+        vm = None
+        params['cpu_model'] = test_model
+        logging.info('Start vm with cpu model %s' % test_model)
+        try:
+            env_process.preprocess_vm(test, params, env, vm_name)
+            vm = env.get_vm(vm_name)
+            output = vm.process.get_output()
+            if warning_text not in output:
+                test.fail("Qemu should output warning for lack flags"
+                          " while it does not.")
+        except Exception as e:
+            if boot_expected == 'no':
+                logging.info('Expect vm boot up failed when enforce is set.')
+                if warning_text not in str(e):
+                    raise
+            else:
+                raise
+        else:
+            if boot_expected == 'no':
+                test.fail('The vm should not boot successfully'
+                          ' when cpu enforce mode is on')
+        finally:
+            if vm and vm.is_alive():
+                vm.destroy()
+
+    vm_name = params['main_vm']
+    old_cpu_models = params['old_cpu_model']
+    new_cpu_models = params['new_cpu_model']
+    warning_text = params.get('warning_text')
+    boot_expected = params.get('boot_expected', 'yes')
+    host_cpu_model = domcapability_xml.DomCapabilityXML().get_hostmodel_name()
```

Comment:
Hi, there will be an error if the host has no libvirt installed. Seems we won't install libvirt default.